### DISCUSSION TITLE
PLC's Connecting Randomly

### DISCUSSION THREAD

**Discussion by EldenZuken** [MikaylaFischler/cc-mek-scada#251] [General]
I have now having a issue connecting all 4 PLC's to the Supervisor. Only one out of the 4 PLC's are connecting even when i turn the Supervisor off and on again. All 4 PLCS show their own units. All of them show the network as green. Any ideas? (update) The PLC's are all connecting and disconnecting from the Supervisor one will connect then disconnect then the next will connect then disconnect it cycles though 1-4 then restarts and cycles though it again.(update again)I changed the listen port # for each one then they all started working!

**Comment by MikaylaFischler** [comment]
Yep, sounded like a listening port issue. #225 is going to be in the next beta update today or tomorrow, which will require config file changes reducing the channel count down to 5 channels (PLC, RTU, Supervisor, Coordinator, Pocket). Each PLC will then use the same channel without issues, as identification will use the computer's ID.
